From 796f4174f7e5b1b990840112ab4d559f685e64f8 Mon Sep 17 00:00:00 2001
From: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
Date: Wed, 29 May 2019 14:25:58 +0530
Subject: [PATCH] [BT-HFP] Disable Wideband Support for Q apl_nuc

intel connectivity chip does not have the WBS codec support
hence disabling it in the stack.

Change-Id: Ib2485b0614d38f062a7bf95c9175f2c89d2a83cf
Tracked-On: OAM-84015
Signed-off-by: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
---

diff --git a/bta/hf_client/bta_hf_client_at.cc b/bta/hf_client/bta_hf_client_at.cc
index 5bcc68e..8e71bc0 100644
--- a/bta/hf_client/bta_hf_client_at.cc
+++ b/bta/hf_client/bta_hf_client_at.cc
@@ -1647,7 +1647,7 @@
 
   APPL_TRACE_DEBUG("%s", __func__);
 
-  buf = "AT+BAC=1,2\r";
+  buf = "AT+BAC=1\r";
 
   bta_hf_client_send_at(client_cb, BTA_HF_CLIENT_AT_BAC, buf, strlen(buf));
 }
